src/code/HTTPRequestHandler/HTTPRequestHandler.test.cpp

```diff
@@ -0,0 +1,31 @@
+
+#include <HTTPRequestHandler/HTTPRequestHandler.hpp>
+#include <gtest/gtest.h>
```

Comment:
no tests?

Reply:
I didnt do tests because i needed sockets and file descriptors and i felt it was not worth it but i can do some